enumeration.py

```diff
@@ -39,3 +39,42 @@ class RoundAdvantage(IntEnum):
     TIME = 0
     BOT_1 = 1
     BOT_2 = 2
+
+
+@unique
+class GamesListFileField(Enum):
```

Comment:
`GameListFileFields` will be more accurate

Reply:
According to this post
https://stackoverflow.com/questions/15755955/naming-of-enums-in-java-singular-or-plural

and I agree with it enums should have singular name.

Reply:
ok